e2e-tests/gef/cypress/e2e/facility/review-ukef-approval/review-ukef-approval-with-issued-and-unissued-facilities-MIA-and-resubmit-to-ukef.spec.js

```diff
@@ -21,17 +21,19 @@ import coverStartDate from '../../pages/cover-start-date';
 import applicationDetails from '../../pages/application-details';
 import applicationActivities from '../../pages/application-activities';
 
+const facilityEndDateEnabled = Number(Cypress.env('GEF_DEAL_VERSION')) >= 1;
+const { unissuedCashFacility, unissuedContingentFacility, unissuedCashFacilityWith20MonthsOfCover } = multipleMockFacilities({ facilityEndDateEnabled });
+const issuedCashFacilityWithCoverDateConfirmed = anIssuedCashFacilityWithCoverDateConfirmed({ facilityEndDateEnabled });
+
 const { format } = require('date-fns');
 
 let dealId;
 let token;
 let facilityTwoId;
 
-const unissuedFacilitiesArray = [MOCK_FACILITY_ONE, MOCK_FACILITY_THREE, MOCK_FACILITY_FOUR];
+const unissuedFacilitiesArray = [unissuedCashFacility, unissuedContingentFacility, unissuedCashFacilityWith20MonthsOfCover];
```

Comment:
Could/should this be in `multipleMockFacilities`? As repeated a lot

Reply:
Either looks good to me @MarRobSoftwire 

RE reliance/assumption on it being a cash facility, if this becomes an issue we could conditionally create the array with a facility type flag or maybe `unissuedCashFacilities` / `facilities.cash.unissued` or something like that

Reply:
I think assuming it's a cash facility is inherently an issue & the reason why I moved away from the non-descriptive names (like `MOCK_FACILITY_ONE`). I think in this case there's some benefit to making sure things are clear, over making code the mocks DRY.

Reply:
OK, happy with this as is. Don't need to overcomplicate.

Seems to make sense for the `unissuedFacilitiesArray` to be defined in individual tests - whilst there is some repetition, they are not all the same and I think a single `unissuedFacilitiesArray` instance could potentially confuse devs in the future.